cal/cal/templates/home_logged_in.html

```diff
@@ -98,7 +98,7 @@ <h4 class="well">Tags - Cumulative</h4>
             </th>
           </thead>
           <tbody>
-            <tr ng-repeat="tag in tags | orderBy:orderByField:reverseSort">
```

Comment:
Wait to merge until I can figure this out, looks like orderBy doesn't work for dictionaries:
http://stackoverflow.com/questions/14330878/orderby-not-working-with-dict-syntax-on-ng-repeat